### ISSUE TITLE
[Question][Help Wanted]: response from authCallback that token is invalid immediately after a validated response

### ISSUE THREAD

**Issue by svagatt** [damienbod/angular-auth-oidc-client#1521]
**What Version of the library are you using?**
...14.1.4

**Question**
This is the config data, 
`export const webAppConfig =  {
    authority: authUrl,
    authWellknownEndpointUrl: authUrl,
    redirectUrl: 'http://localhost:4200/login',
    postLogoutRedirectUri:'http://localhost:4200',
    responseType: 'id_token',
    logLevel: LogLevel.Debug,
    scope: 'openid',
    clientId: 'public-client',
    silentRenew: true,
    useRefreshToken: true
};`
and the app route,
` {
    path: 'login',
    component: LoginPage,
  },
  {
    path: '',
    redirectTo: 'login',
    pathMatch: 'full',
  },
  {
    path: '**',
    redirectTo: 'tabs/home'
  }`,
basically the entry point to the app is the login page where checkAuth() method is being observed,
```
 this.authService.checkAuth().subscribe(async (result: LoginResponse) => {
      console.log('check auth fired');
      console.log(result);
      if (result.isAuthenticated) {
        await Preferences.set({ key: 'token', value: result.accessToken });
        this.router.navigate(['tabs/home']);
      } else{
        console.log('user not logged in');
      }
```
The result I get in my console is:
 [DEBUG] 0-public-client - Working with config '0-public-client' using https://redacted url
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - currentUrl to check auth with:  http://localhost:4200/login#state=redacted&session_state=redacted&id_token= redacted
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - BEGIN callback, no auth data
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - AuthResult '{
  "state": redacted
  "session_state": redacted
  "id_token": redacted
}'.
 AuthCallback created, begin token validation angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - Getting signinkeys from  https://certs-url
[DEBUG] 0-public-client - validate id token iat max offset 3000 < 120000
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - Has idToken expired: false --> expires in 12:57 , 2:18:08 PM > 2:05:11 PM
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - authCallback token(s) validated, continue
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - authCallback token(s) invalid
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - storing the accessToken ''
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - authCallback idToken flow with accessToken 
angular-auth-oidc-client.mjs:4197
[DEBUG] 0-public-client - checkAuth completed - firing events now. isAuthenticated: false
`{isAuthenticated: false, userData: {…}, accessToken: null, idToken: null, configId: '0-public-client'}
accessToken: null
configId: "0-public-client"
idToken: null
isAuthenticated: false
userData: {exp: 1661861888, iat: 1661860988, auth_time: 1661858064, jti: '19fc736a-9181-40ee-a846-c1fb19f3ebd4', iss: 'https://redacted', …}
[[Prototype]]: Object`

**Why do I have an invalid token immediately after token's being validated**, also a null token and id token in login response even though the authcallback is completed and userData is available, would really appreciate if someone could help me, thanks

**Comment by walter-psjr** [comment]
Hi, today I updated a project angular 13 + angular-auth-oidc-client 13 to angular 14 and angular-auth-oidc-client 14.1.4 and started having this problem, with the application getting into an infinite login loop. I already had the maxIdTokenIatOffsetAllowedInSeconds=18000 setting and I realized that if I remove this setting then everything works normally. The strange thing is that in version 13 this configuration with this value was working and now it doesn't.

Does anyone know if any rules have changed regarding the value that can be set? Also, would there be any problem in disabling this idToken offset check? We've had several users with problems because their clocks were late.
